### ISSUE TITLE
Set headers for uploaded files

### ISSUE THREAD

**Issue by jswetzen** [google-github-actions/upload-cloud-storage#22]
### TL;DR
I'm using this command line at the moment:
```gsutil -h "Content-Type:text/xml" -h "Cache-Control:public, max-age=600" cp file.ext gs://bucket-name```
I would like to use this action but don't see how I can without the ability to set headers. Enabling headers would be nice.

### Design

**Action YAML**

```yaml
steps:
  - id: upload-files
    uses: google-github-actions/upload-cloud-storage@main
    with:
      path: /path/to/folder
      destination: bucket-name
      headers:
        - "Content-Type:text/xml"
        - "Cache-Control:public, max-age=600"
```

**Comment by jakub-bacic** [comment]
Unfortunately, GH actions does not support complex values as action inputs, so in order to pass a list we need to be a little bit more creative. I think the most common approach is to use YAML multiline strings:
```yaml
steps:
  - id: upload-files
    uses: google-github-actions/upload-cloud-storage@main
    with:
      path: /path/to/folder
      destination: bucket-name
      headers: |-
        Content-Type:text/xml
        Cache-Control:public, max-age=600
```

I've opened PR that adds support for it.

**Comment by bharathkkb** [comment]
fixed via #42
